src/interfaces/IProposalTypesConfigurator.sol

```diff
@@ -26,6 +33,21 @@ interface IProposalTypesConfigurator {
         uint16 approvalThreshold;
         string name;
         address module;
+        bytes32[] txTypeHashes;
```

Comment:
`txTypeHashes` is an array of function selectors? do they include the address? i think it would be good to have some natspec comments on how `txTypeHashes` should be computed

Reply:
applies to parameters in `Scope` struct as well